src/interfaces.d.ts

```diff
@@ -111,6 +111,11 @@ export interface WNode {
 
 export type DNode = HNode | WNode | string | null;
 
+export interface PropertyChangeRecord {
```

Comment:
I think in https://github.com/dojo/widgets/pull/252, you used the past tense with `Changed`. Should this be `Change` or `Changed`? I personally prefer the former.